### PR TITLE
f-tabs@0.1.0 - generated output

### PR DESCRIPTION
**Pull request by matthewhardern**
Output from generator only. Created component

### CODE REVIEW

File: packages/components/molecules/f-tabs/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@justeat/f-tabs",
+  "description": "Fozzie Tabs – Switchable slots for content&#34;",
```
Comment (ashleynolan) [inline packages/components/molecules/f-tabs/package.json:3]: Is the `&#34` meant to be there?

File: packages/components/molecules/f-tabs/src/components/Tabs.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-tabs']"
+        data-test-id='tabs-component'>
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-tabs/src/components/Tabs.vue:4]: Minor, should be double quotes here. (is this generated like this?)

**Review by kevinrodrigues — APPROVED** [review]

File: packages/components/molecules/f-tabs/package.json (hunk 1 above)
Comment (matthewhardern) [reply packages/components/molecules/f-tabs/package.json:3]: it was generated this way, will change but can we fix this in generator?

File: packages/components/molecules/f-tabs/src/components/Tabs.vue (hunk 2 above)
Comment (matthewhardern) [reply packages/components/molecules/f-tabs/src/components/Tabs.vue:4]: generated this way, I will make the change

File: packages/components/molecules/f-tabs/package.json (hunk 1 above)
Comment (ashleynolan) [reply packages/components/molecules/f-tabs/package.json:3]: Yeh, I'll take a look at that 👍

File: packages/components/molecules/f-tabs/src/components/Tabs.vue (hunk 2 above)
Comment (ashleynolan) [reply packages/components/molecules/f-tabs/src/components/Tabs.vue:4]: I've updated this in the generator now too 👍

**Review by dandel10n — APPROVED** [review]

**Pull request merged**